src/Nethermind/Nethermind.RunnerSpawner/spawner.json

```diff
@@ -1,6 +1,6 @@
 {
-  "ReleasePath": "F:\\IT\\projects\\Crypto\\nethermind_release",
-  "DbBasePath": "F:\\IT\\projects\\Crypto\\db",
+  "ReleasePath": "/home/spetz/Workspace/Nethermind/Runner/",
```

Comment:
:D

Reply:
Since no one's been using it for a while, I put there my path :).